web/src/lib/registration/supabase-storage-service.test.ts

```
@@ -0,0 +1,498 @@
+  let mockUpdate: Mock
+  let mockInsert: Mock
+  let mockDelete: Mock
+
+  beforeEach(async () => {
+    vi.clearAllMocks()
+
+    // Set up mock chain with proper method returns
+    mockSingle = vi.fn().mockResolvedValue({ data: null, error: null })
+    mockEq = vi.fn().mockReturnValue({
+      single: mockSingle,
+      then: <T>(fn: PromiseCallback<T>) => mockSingle.then(fn) // Make eq() thenable for direct resolution
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)